docs/learn/data-availability.md

```diff
@@ -0,0 +1,35 @@
+
+### Pinning
+Pinning provides a more long-lived mechanism of data persistence for commits. Pinning is a process for instructing a Ceramic node to explicitly host (i.e. "pin") the commits for a specific stream. Since commits are stored in IPLD, Ceramic nodes already contain a bundled [IPFS](./glossary.md#ipfs) node which is where this pinning occurs. IPFS nodes can pin all commits for any stream which is accessible over the Ceramic network to which it is connected. Ceramic pinning can also work using an external IPFS node instead of the bundled internal version.
+
+If developers want the easiest way to make their streams persistent beyond a single session and more resilient against data loss, then pinning is the right option. Ceramic nodes can pin an unlimited number of streams. However, note that if only one IPFS node is pinning a given stream and it disappears forever or gets corrupted, then that stream will be lost. Also, if only one node is pinning a stream (and no other Ceramic nodes have it in cache) and that node goes offline, then that stream will be unavailable to others. Therefore, for improved resilience and data availability it is best to have multiple IPFS nodes running in different envitonments pinning the same streams.
+
+> See the [Pinning](../build/pinning.md) guide for instructions on how to pin streams on a Ceramic node.
+
+### Archiving
+Archiving is the most durable, long-lived form of persistence for commits. In addition to caching and pinning, Ceramic developers may also configure their node to connect to an external service for archiving all commits that make up a stream. The exact guarantees provided by archiving differ with each implementation and service provider. For example, archiving to Filecoin provides crypto-economically guaranteed data availability with a pay-as-you-go model, while archiving to Arweave provides crypto-economically guaranteed data availability with a one-time payment model. Conversely, archiving to Amazon S3 provides a simpler model however Amazon cannot guarantee that your data will always be available (for example you could stop paying your bill), but the storage is still more resilient than using pinning and caching only.
+
+## **Stream state**
+In addition to the commit log mentioned above which is stored in IPFS, every stream has a [state](./glossary.md#state) which is not stored in IPFS but rather is collectively tracked, persisted, and made available by all Ceramic nodes that are caching and/or pinning the stream. Both the complete commit history and the state are needed to successfully load and interact with streams.
+
+### Caching
+State caching works the same as commit caching.
+
+### Pinning
+State pinning works the same as commit pinning, except state pinning does not occur on IPFS. State pinning simply occurs in a database internal to the Ceramic node.
+
+### Archiving
```

Comment:
We should probably mention the S3 state store in here somewhere

Reply:
Where would that fit in exactly? @stbrody

Reply:
I think an extra sentence at the end of the Archiving section would be fine. Something like "In the meantime, Ceramic supports archiving state to S3, for an option with more durability and reliability than the local database used by default"

Reply:
Perfect, ty @stbrody. Made that edit. Going to merge to main now